content/docs/guides/design/index.en.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "OSRD's design"
+linkTitle: "OSRD's design"
+weight: 10
+description: Colours, fonts, uses...
```

Comment:
To match french version
```suggestion
description: Colours, fonts, logos, uses...
```

Reply:
Thx! Corrected but in another way: I removed "logo" on the french side. We speak about logos inside another file ;)

Reply:
LGTM, thanks !